src/main/scala/dsptools/tester/DspTester.scala

```diff
@@ -64,15 +64,21 @@ class DspTester[+T <: MultiIOModule](
     incTime(n)
   }
 
+  private def maskedBigInt(bigInt: BigInt, width: Int): BigInt = {
+    val maskedBigInt = bigInt & ((BigInt(1) << width) - 1)
+    maskedBigInt
+  }
+
   override def poke(path: String, value: BigInt): Unit = {
     if (verilogTb) throw DspException("Don't use poke path (String) when printing verilog tb")
     backend.poke(path, value)(logger, dispDsp, dispBase)
   }
 
   override def poke[T <: Element : Pokeable](signal: T, value: BigInt): Unit = {
     // bit-level poke is displayed as unsigned
+    val maskeValue = maskedBigInt(value, signal.widthOption.getOrElse(128))
```

Comment:
typo on the name, should be `maskedValue`

Reply:
Fixed